drivers/timer/hpet.c

```diff
@@ -98,6 +98,14 @@ int z_clock_driver_init(struct device *device)
 	return 0;
 }
 
+void smp_timer_init(void)
```

Comment:
Might want to wrap this inside "#ifdef CONFIG_SMP"

Reply:
Not sure I see why; if it's not referenced it won't get pulled into the link.

Reply:
I was thinking "defined but no used" warning. Come to think of it, it shouldn't matter.